ui/lib/core/addon/helpers/changelog-url-for.js

```diff
@@ -22,8 +22,8 @@ export function changelogUrlFor([version]) {
   const url = 'https://www.github.com/openbao/openbao/blob/main/CHANGELOG.md#';
   if (!version) return url;
   try {
-    // strip the '+prem' from enterprise versions and remove periods
```

Comment:
I think we should continue to strip `+` suffixes. I've been setting `+gitlab.<build>` in my internal builds:

https://gitlab.com/gitlab-org/govern/secrets-management/openbao-internal/-/blob/main/Makefile?ref_type=heads#L8